quantumdb-cli/src/main/java/io/quantumdb/cli/xml/XmlAddColumn.java

```diff
@@ -21,12 +21,19 @@ static XmlOperation convert(XmlElement element) {
 		checkArgument(element.getTag().equals(TAG));
 
 		XmlAddColumn operation = new XmlAddColumn();
-		operation.setTableName(element.getAttributes().get("tableName"));
+		operation.setTableName(element.getAttributes().remove("tableName"));
```

Comment:
Instead of 'get' use 'remove', this is useful to throw an error when an element is not used, therefore the XML is malformed.

Reply:
Uhw, good idea!